core/src/main/java/tech/pegasys/web3signer/core/config/BaseConfig.java

```diff
@@ -49,6 +49,16 @@ public interface BaseConfig {
 
   List<String> getMetricsHostAllowList();
 
+  Boolean isMetricsPushEnabled();
+
+  String getMetricsPushHost();
+
+  Integer getMetricsPushPort();
+
+  Integer getMetricsPushIntervalSeconds();
+
+  String getMetricsPrometheusJob();
+
```

Comment:
nit: `MetricsPushOptions` value object, similar to TlsOptions, can also be used to eliminate need of multiple methods in BaseConfig.